app/partials/navigation.jade

```diff
@@ -12,7 +12,7 @@
     .navbar-collapse.collapse(ng-class="{'in bg-blue' : !navCollapsed}", ng-click="navCollapsed=true")
       ul.nav.navbar-nav.navbar-right(ng-show="status.isLoggedIn")
         li.item(ng-show="settings.btcCurrency")
-          a.flex-important.flex-column(ng-click="openBCmarkets()")
+          a.flex-important.flex-column(ng-click="openBCmarkets()" href="")
```

Comment:
href="" makes the anchor tag focusable, which allows users to tab around and select. Without href="" the anchor tag is skipped over. I'm not sure if angular can put href="" everywhere, but I only added them to places I thought were important, like the dropdown menu in the header, or the dropdown for "accounts" in send.